client/layout/sidebar/style.scss

```diff
@@ -233,13 +233,30 @@
 			.sidebar__expandable-arrow {
 				fill: var( --color-sidebar-menu-hover-text );
 			}
+
+			.count {
+				background-color: transparent;
+				border-color: var( --color-sidebar-menu-selected-text );
+				color: var( --color-sidebar-menu-selected-text );
+			}
 		}
 	}
 
 	.sidebar__expandable-title {
 		flex: 1 1 0;
 	}
 
+	.count {
+		margin-left: 8px;
+		color: var( --color-sidebar-menu-hover-text );
+		background-color: var( --color-sidebar-menu-hover-background );
+		border-color: var( --color-sidebar-menu-hover-background );
+		line-height: 0;
+		padding: 8px 6px;
+		position: absolute;
+		top: 9px;
+	}
+
```

Comment:
Can we consolidate all this count rules with these https://github.com/Automattic/wp-calypso/blob/9e259071948215c8269ff383ef9b2087d55118f6/client/layout/sidebar/style.scss#L144-L158
and probably nest them to `.sidebar {}`?

Also, it seems that can we entirely remove `position` and `top`.

The above will help us avoid this situation where we have quite the same rules clashing and an other counter misplaced.
![](https://cln.sh/vSzTNA+)

Reply:
Yes, please feel free to make those changes. I was unfamiliar with how that file i supposed to be structured and didn't want to step on anyone's toes.

Reply:
Done, I also updated the description with nav-unification disabled status.